### DISCUSSION TITLE
Show rendering status when viewing logs

### DISCUSSION THREAD

**Discussion by jameskuszmaul-brt** [foxglove/community#854] [Q&A]
It would be amazing, especially for the plot and state transition panels, to have some indicator that tells users when they are fully rendered so that you can accurately determine whether a lack of data in a panel is because you need to wait longer or because, e.g., there just isn't data in a requested topic past a certain timepoint.

E.g., in this view, it is not obvious whether the lack of data in the upper left plot panel is because it has not yet rendered or because there is some other issue--in this case, that data is ~10x more frequent than the other panel, and so just hadn't fully loaded yet:
![image](https://github.com/foxglove/community/assets/91565539/8f2c3843-5b02-477b-a176-7232e5239e0f)

**Comment by cfoust** [comment]
This is a great idea!
